subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.h

```diff
@@ -5,6 +5,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define RADIO_CRC_POLYNOMIAL ((0x5bUL) | ((0x06UL) << 8) | ((0x00UL) << 16))
+
+
```

Comment:
Define this in the `subsys/bluetooth/controller/ll_sw/pdu.h` as `PDU_CRC_POLYNOMIAL`

Reply:
Will do